docs/CONTRIBUTING.md

```diff
@@ -14,13 +14,26 @@ If you are a *git ninja*, we feel happy and sad for you at the same time. Your s
 ## Reviews
 Reviews are a great way to get familiar with the codebase. If you would like to start with reviewing just use **[this handy github filter](https://github.com/xbmc/xbmc/pulls?q=is%3Apr+is%3Aopen+review%3Anone)** to see PRs that are in need of one (we have plenty!).
 
-## Pull request guidelines
+## Pull request making guidelines
 * **Create topic branches**. Never ever open a pull request from your master branch. **Ever!**
 * **Code against master branch**. Unless release specific, all Kodi development occurs in the `master` branch. If it needs backporting it can be done after it hits master.
 * **One topic branch per change**. Found something that needs fixing but is unrelated to the current work? Create a new topic branch and open another *pull request*.
 * **Use a coherent commit history**. Make sure each individual commit in your *pull request* is meaningful and organized in logical chunks. Tidy up and squash commits before submitting.
-* **Be descriptive, but concise**. The *pull request* and commits should have a useful title and follow the `[component(s)] Short description...` format. No, `[README.md] Update` is **not** descriptive enough. The *pull request* description should only contain information relevant to the change.
-* **Be patient, be kind, rewind**. We're all volunteers with busy lives but usually very responsive in cataloging *pull requests* and pinging the relevant maintainer(s). Sometimes though, things go unnoticed for longer than usual. Probably due to beer deprivation... or lack thereof. Feel free to nudge us if we forget.
+* **Be descriptive, but concise**. The *pull request* and *commits* should have an useful title and follow the format:
+`[Component(s)] Short description...` or alternative format `Component(s): Short description...`<br/>
```

Comment:
One important thing not many people do is to keep a concise description on the commit message (not the title but the message below). That's pretty much the only way the context is kept in git history without being bound to a specific git platform like github (unlike the PR body, issues, git comments, etc). I would say that if the contribution contains code that need additional explanation (why that change is being done and what others should take into consideration if ever considering changing that piece of code) a proper commit message should be added as well.
I don't see the need to make it mandatory for all commits (only on particular circumstances). Otherwise if we ever move from github to a different platform there is no guarantee that all the other context is kept.

Reply:
modified take a second look if you prefer reword in other way